WordPress/Classes/Models/Notifications/Notification+CoreDataProperties.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import CoreData
+
+
+// MARK: - Notification Core Data Properties
+//
+extension Notification {
```

Comment:
Is there a reason it is split out like this? I find this actually more confusing than it being in `Notification` itself.